### PR TITLE
Added requote text plugin

### PR DESCRIPTION
**Pull request by FayasNoushad**
For requote text for URL.

Like :- `Hi Hello` to `Hi%20Hello`

### CODE REVIEW

File: plugins/requote.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+@userge.on_cmd("requote", about={
+    'header': "Requote Text",
+    'description': "get requoted text from a normal text",
+    'usage': "{tr}requote [text]"})
+async def requote(update: Message):
+    if len(update.message.text.split()) <= 1:
+        text = "Add requote text too."
+    else:
+        text = requote_uri(update.message.text.split(" ", 1)[1])
+    await update.message.edit_text(
```
Comment (rking32) [inline plugins/requote.py:16]: ...

**Review by rking32 — COMMENTED** [review]
thanks

File: requirements.txt [hunk 2]
```diff
@@ -13,12 +14,12 @@ humanize
 justwatch
 lottie
 newspaper3k
+psutil
 pydub
 pylast
 pyfiglet
 python-arq
 qrcode
+requests
```
[the remark this answers is not in the document]
Comment (FayasNoushad) [reply requirements.txt:23]: Ok

File: plugins/requote.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+from requests.utils import requote_uri
+from userge import userge, Message
+
+
+@userge.on_cmd("requote", about={
+    'header': "Requote Text",
+    'description': "get requoted text from a normal text",
+    'usage': "{tr}requote [text]"})
+async def requote(update: Message):
+    if len(update.message.text.split()) <= 1:
```
[the remark this answers is not in the document]
Comment (FayasNoushad) [reply plugins/requote.py]: Sorry, I used in CallbackQuery style. Sorry, its my mistake.

File: requirements.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (rking32) [reply requirements.txt:23]: so can you remove it ?

**Comment by FayasNoushad** [comment]
> thanks

Always welcome

**Comment by FayasNoushad** [comment]
> > thanks
> 
> Always welcome

Can you merge?

**Pull request merged**